news/5119.bugfix

```diff
@@ -0,0 +1 @@
+Fix svg icon in toolbar.jsx. @MoazMirza-13
```

Comment:
```suggestion
Replace user icon with settings icon, and update the text "username" to "Settings" in the lower-left for authenticated users. @MoazMirza-13
```

Reply:
@stevepiercy can you help me out ?
i tried to find this "username" in the codebase but i am not able to find it.